src/main/java/hw3/service/UserService.java

```diff
@@ -0,0 +1,9 @@
+package hw3.service;
+
+import hw3.dto.UserRegistrationDto;
+import hw3.dto.UserResponseDto;
+
+public interface UserService {
+    UserResponseDto registerUser(UserRegistrationDto userRegistrationDto);
+    UserResponseDto getUserById(Long userId);
```

Comment:
You are already working with users in UserService, so method name could be:
```suggestion
    UserResponseDto getById(Long userId);
```